raaghu-components/src/rds-comp-feeds/rds-comp-feeds.tsx

```diff
@@ -8,6 +8,7 @@ export interface Item {
     feedIcon?: string;
     imageUrl?: string;
     description: string;
+    descriptionShort?: string;
```

Comment:
@prakashmhaske  this property is not used in interface, why this is added?

Reply:
Removed this property as this is not in use.